docs-src/references/strongly-typed-errors/cause-unhandled-update.md

```diff
@@ -0,0 +1,15 @@
+---
+id: cause-unhandled-update
+title: Cause Unhandled Update
+description: Explanation for Cause Unhandled Update error message.
+sidebar_label: Cause Unhandled Update
+tags:
+  - errors
+  - strongly-typed
+---
+
+`UnhandledUpdate` occurs when a Workflow Update is received by the server while a Workflow Task is being processed on a Worker producing a command that would cause the Workflow to transition to a closed state.
```

Comment:
```suggestion
`UnhandledUpdate` occurs when a [Workflow](/concepts/what-is-a-workflow) update is received by the Temporal Server while a Workflow Task being processed on a Worker produces a Command that would cause the Workflow to transition to a closed state.
```